index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+
+</head>
+<body>
+    <nav id="navbar" class="nav">
+        <ul class="nav-list">
+          <li>
+            <a href="#welcome-section">About</a>
+          </li>
+          <li>
+            <a href="#projects">Work</a>
+          </li>
+          <li>
+            <a href="#contact">Contact</a>
+          </li>
+        </ul>
+      </nav>
+   <div id="welcome-section" width="100%" heigth="100%">
+    <h1>Welcome to my personal portfolio</h1>
+    </div> 
+    <section id="projects" >
+        <div class="projects-grid">
+        <div class="project-tile">
+            <a href="https://en.wikipedia.org/wiki/Website" >
+                <h2>My first Project</h2>
+                <img src="https://upload.wikimedia.org/wikipedia/commons/8/87/United_States_Antarctic_Program_website_from_2018_02_22.png" alt="my first project">
+            </a>
+            </div>
+
+            <div class="project-tile">
+                <a href="https://en.wikipedia.org/wiki/Website" >
+                    <h2>My second Project</h2>
+                    <img src="https://upload.wikimedia.org/wikipedia/commons/8/87/United_States_Antarctic_Program_website_from_2018_02_22.png" alt="my second project">
+                </a>
+            </div>
+        </div>
+    </section>
+    
+<div id="contact">
+    <a href="https://wwww.github.com/Edcaly" id="profile-link" target="_blank">Visit me on Github</a>
+
+</div>
+
+    
+    <footer>
+        <p>
+          ** New prototype of a portafolio of of microverse projects
+        </p>
+        <p>
+          &copy; Created by
+          <a href="https://github.com/lordksix" target="_blank"
+            >lordksix and Edcaly <i class="fab fa-free-code-camp"></i
+          ></a>
```

Comment:
Good job setting up live and useful links using <a> tag :+1: , however, whenever **target="_blank"** it is used, it is necessary to use **rel="noopener"**. More information can be found [here](https://webhint.io/docs/user-guide/hints/hint-disown-opener/). :eyeglasses: